herddb-thirdparty/openjpa-test/pom.xml

```diff
@@ -41,7 +41,7 @@
         <dependency>
             <groupId>org.apache.openjpa</groupId>
             <artifactId>openjpa</artifactId>
-            <version>3.1.1</version>
+            <version>3.1.3-SNAPSHOT</version>
```

Comment:
this can be seen a problematic for a release, but it is needed in order to see the test work.
if we want to cut a release we must revert back to 3.1.2 and comment parts of the test
I hope that with @rmannibucau 's help we could release OpenJPA 3.1.3 soon